core/sql/exp/ExpLOBaccess.cpp

```diff
@@ -552,8 +552,9 @@ Ex_Lob_Error ExLob::statSourceFile(char *srcfile, Int64 &sourceEOF)
    if (srcType == HDFS_FILE)
      {
        hdfsFile sourceFile = hdfsOpenFile(fs_,srcfile,O_RDONLY,0,0,0);   
```

Comment:
Is this where we are expecting errno to be set? We could capture its value here.